build-script/code-serving-doc-parser/scripts/init.sh

```
@@ -0,0 +1,89 @@
+  echo "[init.sh] BUILD_COMMAND completed."
+else
+  # 디폴트: requirements.txt 있으면 pip install (--find-links로 packages/도 참조).
+  REQ_FILE="$DESTINATION/requirements.txt"
+  if [ -f "$REQ_FILE" ]; then
+    FIND_LINKS=""
+    if [ -d "$DESTINATION/packages" ]; then
+      FIND_LINKS="--find-links $DESTINATION/packages"
+    fi
+    echo "[init.sh] requirements.txt detected, installing packages..."
+    # pip 은 PATH 상 /app/.venv/bin/pip (base deps 와 동일 venv) 로 해석된다.
+    pip install $PIP_OPTS -r "$REQ_FILE" $FIND_LINKS 2>&1 || echo "[init.sh] WARNING: pip install failed"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)